python/lsst/pipe/tasks/propagateVisitFlags.py

```diff
@@ -165,28 +166,23 @@ def run(self, butler, coaddSources, ccdInputs, coaddWcs):
         @param[in,out] coaddSources  Source catalog from the coadd
         @param[in] ccdInputs  Table of CCDs that contribute to the coadd
         @param[in] coaddWcs  Wcs for coadd
+        @param[in] visitCatalogs List of loaded source catalogs for each input ccd in
+                                 the coadd, this is used instead of this method loading
+                                 in the catalogs itself
```

Comment:
-> "the coadd. *If provided*, this is used instead of this method loading in the catalogs itself."